examples/nodejs-ext/CommunityToolkit.Aspire.Hosting.NodeJS.Extensions.AppHost/CommunityToolkit.Aspire.Hosting.NodeJS.Extensions.AppHost.csproj

```diff
@@ -19,6 +19,7 @@
 
   <Target Name="CleanNodeModules" AfterTargets="Build">
     <Exec WorkingDirectory="../vite-demo" Command="rm -r node_modules" Condition="Exists('../vite-demo/node_modules')" />
+    <Exec WorkingDirectory="../deno-demo" Command="rm -r node_modules" Condition="Exists('../deno-demo/node_modules')" />
```

Comment:
Should this be removed now?

```suggestion
```

Reply:
I don't think that all can be removed, but Deno for sure can be removed. From the testing fixture standpoint I can see why you would want these node modules folders to be cleared, so that you're verifying  installation where that behavior is part of something like `AddViteApp`.  Thoughts?

That aside, I kept running into issues with this MSBuild target when rebuilding this project. This resulted in me manually running these `rm -rf` commands. Maybe they need to be replaced with `Delete` tasks?

Reply:
`Delete` tasks seemed to do the trick.